roles/arteria-sequencing-report-ws/tasks/install_config.yml

```diff
@@ -21,7 +21,7 @@
 
 - name: Deploy arteria-delivery wrapper
   template:
-    src: arteria-sequencing-report_wrapper.sh.j2
+    src: arteria_sequencing_report_wrapper.sh.j2
```

Comment:
In this wrapper, the nextflow executable is sourced from the module system, rather than using the nextflow that is deployed. Is this intentional?

Reply:
Errr, good question. I don't know - @Aratz ?